go.mod

```diff
@@ -23,17 +23,17 @@ require (
 	go.uber.org/zap v1.26.0
 	golang.org/x/sync v0.4.0
 	google.golang.org/grpc v1.59.0
-	k8s.io/api v0.27.7
-	k8s.io/apimachinery v0.27.7
-	k8s.io/client-go v0.27.7
+	k8s.io/api v0.28.3
```

Comment:
Can we do that, regarding K8s min version?

Reply:
We use `replace` field as https://github.com/nak3/serverless-operator/blob/fix-ut/go.mod#L200 so this line is ignored.